workflows/raredisease.nf

```diff
@@ -135,6 +135,10 @@ workflow RAREDISEASE {
         exit 1, 'Input samplesheet not specified!'
     }
 
+    if (params.variant_caller.equals("sentieon") && !params.ml_model) {
+        exit 1, 'Machine learning model not specified!'
```

Comment:
Is the ml model a requirement of running sentieon or is it more that it's not very smart to run without it?

Reply:
Its more of the latter. From what I can tell from reading the documentation, using the ml model is supposed to give much better results that one can filter on, and there is no separate license to use the model. So I don't see why anyone would run without it. Having said that, I figured we can tackle that problem if an user raises that issue at some point.